### ISSUE TITLE
Tried on X9D 2019+ and won't work

### ISSUE THREAD

**Issue by Billy60** [EdgeTX/flasher#39]
Good evening. 
I tried to upgrade my Frsky X9d2019+ from its current OTX2.3 ans using the SDCard transfer the only result I got was:
* Edge Fw is on the flashing choice list on the SDCard
* OTX also.
* I installed Edge
* Then Edge reported installed ok but the OTX kept running. I had installed the Edge Bootloader...
* After a few tries, I got the Edge logo but it was OTX running. Or else there are no differences betw E and O !
* All my models were gone. No conversion whatsoever.
* So I decided to go back to OTX from the saved SDcard.
* Now OTX runs but the bootloader is still Edge's ! it stays there and the OTX one does not install.
Lots of strange things. Since I am fluent in OTX, what is the purpose on a X9D ?

Have a good evening

**Comment by Billy60** [comment]
Good afternoon
Well yes all the files and folders were on the SD.
I'm sorry, I have no screen shot from the loader. I have put it back to OTX, it is complete again and will not try again to improve since my OTX is running fine.
Anyway I am not sure Edge is so useful on a non tactile screen like my X9D2019 has.
Thanks for trying !

**Comment by pfeerick** [comment]
I'f the bootloader is remaining, do I understand you are using the firmware to update the bootloader, and then bootloader to update the firmware?

If you got the edgetx logo to appear, you are running edgetx - the firmware controls the logo (sound comes from the as card pack, naturally). On black and white radios, the UI has only had minor changes (don't fix won't ain't broke in effect). That may change starting in ,2.7. 

So far, reasons to change to edgetx for x9d would be an extra timer condition, one it for faster expresslrs, some extra lua script APIs and with 2.6, yaml configuration file format. Probably missed something. But, yes, focus has been on colorlcd (not specifically touch) but we are starting planning on tweaking B&W UI ...

**Comment by Billy60** [comment]
Ah, so then I have been running with Edge, while I was thinking it was OTX.
I had followed th installing video I found on Youtube, which has been added to the Edge Wiki (did I read).
It says the Edge Bootloader can handle both Otx and Edge. Now I have OTX running but the Bootloader is from Edge. I cannot get the OTX bootloader to come back.
But it doe not matter, my radio works fine now, with OTX. 
I will wait for further upgrades in ergonomy to move to Edge on this BW radio. Meanwhile it is not so necessary.
OTX egonomy is really a  pain in the neck, a steep learning curve. But now its ok for me !
Keep on upgrading Edge, the day it is as good as Ethos then it will be useful in my opinion.
